### PR TITLE
- add function to create categories base on page title

### PR DESCRIPTION
**Pull request by hemerem-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/views/common/index/home.njk [hunk 1]
```diff
@@ -86,7 +86,12 @@
       </svg>
     </button>
     <p>
-      <a href="https://www.gov.uk/guidance/register-as-an-authorised-agent" data-event-id="Read the guidence">{{ i18n.readThe }}</a>.
+      <a href="https://www.gov.uk/guidance/register-as-an-authorised-agent" data-event-id="Read the guidence" id="Read the guidence-id">{{ i18n.readThe }}</a>.
```
Comment (cbroadley-ch) [inline src/views/common/index/home.njk:89]: Spelling mistake here on word "guidance"

File: src/views/common/index/home.njk [hunk 2]
```diff
@@ -86,7 +86,12 @@
...
     </p>
   </form>
+          <script>
+        trackEventBasedOnPageTitle("START NOW-id", "click-button", "START NOW");
+        trackEventBasedOnPageTitle("Read the guidence-id", "click-link", "Read the guidence");
```
Comment (cbroadley-ch) [inline src/views/common/index/home.njk:94]: Same as above

File: src/views/partials/__footer.njk [hunk 3]
```diff
@@ -26,25 +25,40 @@
...
+    });
+  }
 
-<script src="{{ cdnHost }}/javascripts/govuk-frontend/v4.6.0/govuk-frontend-4.6.0.min.js"></script>
-<script src="{{ cdnHost }}/javascripts/app/cookie-consent/cookie-consent-1.0.0.js"></script>
-<script src="{{ cdnHost }}/javascripts/app/cookie-consent/piwik-only-cookie-consent.js"></script>
-<script src="{{ cdnHost }}/javascripts/vendor/jquery-3.3.1.min.js"></script>
-<script>window.GOVUKFrontend.initAll()</script>
+</script>
+<!-- End Matomo Code --></footer><script src="{{ cdnHost }}/javascripts/govuk-frontend/v4.6.0/govuk-frontend-4.6.0.min.js"></script><script src="{{ cdnHost }}/javascripts/app/cookie-consent/cookie-consent-1.0.0.js"></script><script src="{{ cdnHost }}/javascripts/app/cookie-consent/piwik-only-cookie-consent.js"></script><script src="{{ cdnHost }}/javascripts/vendor/jquery-3.3.1.min.js"></script><script>
```
Comment (cbroadley-ch) [inline src/views/partials/__footer.njk:63]: Might be more legible to separate the scripts onto individual lines here

**Review by cbroadley-ch — APPROVED** [review]
LGTM

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Pull request merged**